packages/components/src/internal/components/ontology/OntologyTabs.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react';
+import { Col, Nav, NavItem, Row, Tab } from 'react-bootstrap';
+
+import { OntologyTreePanel } from './OntologyTreePanel';
+import { OntologyModel } from './models';
+
+export const enum OntologyPickerTabs {
+    ONTOLOGY_TREE_TAB = 'ontologytree',
+    ONTOLOGY_SEARCH_TAB = 'search',
+}
+
+interface OntologyTabsProps {
+    model: OntologyModel;
+}
+
+export const OntologyTabs: FC<OntologyTabsProps> = props => {
```

Comment:
I know this component is a WIP until we get further designs on the search phase 2 piece. I might even suggest removing this all together and just most the `<OntologyTreePanel/>` component up into OntologyBrowserPanel.tsx for now.